ide/xml.text/apichanges.xml

```diff
@@ -91,9 +91,16 @@ is the proper place.
             <compatibility binary="compatible" source="incompatible" semantic="compatible" addition="yes" deprecation="yes" deletion="yes"/>
```

Comment:
New entry in `apichanges.xml` is needed warning everyone that this is semantic and binary incompatible change!